[33mWARNING [0m juju.client.connection:connection.py:858 unexpected facade SSHServer received from the controller
[32mINFO    [0m pytest_operator.plugin:plugin.py:753 Adding model github-pr-5f6fe-microk8s:test-acme-aqjo on cloud microk8s
[33mWARNING [0m juju.client.connection:connection.py:858 unexpected facade SSHServer received from the controller
[33mWARNING [0m juju.client.connection:connection.py:858 unexpected facade SSHSession received from the controller
[33mWARNING [0m juju.client.connection:connection.py:858 unexpected facade SSHTunneler received from the controller
[32mINFO    [0m juju.model:__init__.py:2301 Deploying local:vault-k8s-0
[32mINFO    [0m juju.model:__init__.py:2301 Deploying ch:amd64/noble/self-signed-certificates-263
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  vault-k8s/0 [allocating] waiting: installing agent
  vault-k8s/1 [allocating] waiting: installing agent
  vault-k8s/2 [allocating] waiting: installing agent
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  self-signed-certificates/0 [allocating] waiting: installing agent
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  vault-k8s/0 [idle] waiting: Waiting for CA certificate to be accessible in the charm
  vault-k8s/1 [allocating] waiting: agent initialising
  vault-k8s/2 [allocating] waiting: agent initialising
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  self-signed-certificates/0 [idle] active: 
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  vault-k8s/1 [idle] blocked: Please initialize Vault or integrate with an auto-unseal provider
  vault-k8s/2 [idle] blocked: Please initialize Vault or integrate with an auto-unseal provider
[32mINFO    [0m tests.integration.helpers:helpers.py:215 Vault initialized
[32mINFO    [0m tests.integration.vault:vault.py:94 Unsealed vault unit: https://10.1.36.211:8200.
[32mINFO    [0m tests.integration.vault:vault.py:76 Vault unit is unsealed.
[32mINFO    [0m tests.integration.vault:vault.py:76 Vault unit is unsealed.
[32mINFO    [0m tests.integration.vault:vault.py:94 Unsealed vault unit: https://10.1.36.216:8200.
[32mINFO    [0m tests.integration.vault:vault.py:76 Vault unit is unsealed.
[32mINFO    [0m tests.integration.vault:vault.py:94 Unsealed vault unit: https://10.1.36.217:8200.
[32mINFO    [0m tests.integration.vault:vault.py:76 Vault unit is unsealed.
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  vault-k8s/0 [idle] active: 
  vault-k8s/1 [idle] blocked: Please authorize charm (see `authorize-charm` action)
  vault-k8s/2 [idle] blocked: Please unseal Vault
[32mINFO    [0m tests.integration.helpers:helpers.py:246 Charm authorized
[32mINFO    [0m juju.model:__init__.py:3254 Waiting for model:
  vault-k8s/0 [executing] active: 
  vault-k8s/1 [executing] blocked: Please authorize charm (see `authorize-charm` action)
  vault-k8s/2 [idle] blocked: Please unseal Vault
  self-signed-certificates/0 [idle] active:
[32mINFO    [0m pytest_operator.plugin:plugin.py:937 Model status:

Model           Controller                Cloud/Region        Version  SLA          Timestamp
test-acme-aqjo  github-pr-5f6fe-microk8s  microk8s/localhost  3.6.7    unsupported  17:59:43Z

App                       Version  Status  Scale  Charm                     Channel   Rev  Address         Exposed  Message
self-signed-certificates           active      1  self-signed-certificates  1/stable  263  10.152.183.58   no       
vault-k8s                          active      3  vault-k8s                             0  10.152.183.208  no       

Unit                         Workload  Agent  Address      Ports  Message
self-signed-certificates/0*  active    idle   10.1.36.209         
vault-k8s/0*                 active    idle   10.1.36.211         
vault-k8s/1                  active    idle   10.1.36.216         
vault-k8s/2                  active    idle   10.1.36.217         

[32mINFO    [0m pytest_operator.plugin:plugin.py:943 Juju error logs:

unit-self-signed-certificates-0: 17:59:09 ERROR unit.self-signed-certificates/0.juju-log At least 1 matching relation ID not found with the relation name 'send-ca-cert'
unit-self-signed-certificates-0: 17:59:19 ERROR unit.self-signed-certificates/0.juju-log certificates:1: At least 1 matching relation ID not found with the relation name 'send-ca-cert'
unit-self-signed-certificates-0: 17:59:20 ERROR unit.self-signed-certificates/0.juju-log certificates:1: At least 1 matching relation ID not found with the relation name 'send-ca-cert'
unit-self-signed-certificates-0: 17:59:23 ERROR unit.self-signed-certificates/0.juju-log certificates:1: At least 1 matching relation ID not found with the relation name 'send-ca-cert'
unit-self-signed-certificates-0: 17:59:26 ERROR unit.self-signed-certificates/0.juju-log certificates:1: At least 1 matching relation ID not found with the relation name 'send-ca-cert'
unit-self-signed-certificates-0: 17:59:33 ERROR unit.self-signed-certificates/0.juju-log At least 1 matching relation ID not found with the relation name 'send-ca-cert'

[32mINFO    [0m pytest_operator.plugin:plugin.py:1049 Resetting model test-acme-aqjo...
[32mINFO    [0m pytest_operator.plugin:plugin.py:1038    Destroying applications vault-k8s
[32mINFO    [0m pytest_operator.plugin:plugin.py:1038    Destroying applications self-signed-certificates
[32mINFO    [0m pytest_operator.plugin:plugin.py:1054 Not waiting on reset to complete.
[32mINFO    [0m pytest_operator.plugin:plugin.py:1025 Forgetting model main...